src/PageHeader/PageHeader.tsx

```diff
@@ -0,0 +1,387 @@
+      {children}
+    </Box>
+  )
+}
+
+function displayResponsively<T, F>(value: T, fallback: F): BetterSystemStyleObject {
+  if (isResponsiveValue(value)) {
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    const responsiveValue = value as Extract<T, ResponsiveValue<any>>
+
+    return {
+      [`@media screen and (max-width: 768px)`]: {
+        display: 'narrow' in responsiveValue ? (responsiveValue.narrow ? 'none' : 'flex') : fallback ? 'none' : 'flex',
+      },
+      [`@media screen and (min-width: 768px)`]: {
+        display:
+          'regular' in responsiveValue ? (responsiveValue.regular ? 'none' : 'flex') : fallback ? 'none' : 'flex',
+      },
+      [`@media screen and (min-width: 1440px)`]: {
+        display: 'wide' in responsiveValue ? (responsiveValue.wide ? 'none' : 'flex') : fallback ? 'none' : 'flex',
+      },
+    }
+  } else {
+    return {display: fallback ? 'none' : 'flex'}
+  }
+}
+
+// PageHeader.ContextArea : Only visible on narrow viewports by default to provide user context of where they are at their journey. `visible` prop available
+// to manage their custom visibility but consumers should be careful if they choose to hide this on narrow viewports.
+// PageHeader.ContextArea Sub Components: PageHeader.ParentLink, PageHeader.ContextBar, PageHeader.ContextAreaActions
+// ---------------------------------------------------------------------
+const ContextArea: React.FC<React.PropsWithChildren<PageHeaderProps>> = ({
+  children,
+  hidden = hiddenOnRegularAndWide,
+  sx = {},
+}) => {
+  const contentNavStyles = {
+    flexDirection: 'row',
+    alignItems: 'center',
+    gap: get('space.2'),
+    order: REGION_ORDER.ContextArea,
+    ...displayResponsively(hidden, false),
+  }
+  return <Box sx={merge<BetterSystemStyleObject>(contentNavStyles, sx)}>{children}</Box>
+}
+// adopted from React.AnchorHTMLAttributes
+type LinkProps = {
+  download?: string
+  href?: string
+  hrefLang?: string
+  media?: string
+  ping?: string
+  rel?: string
+  target?: string
+  type?: string
+  referrerPolicy?: React.AnchorHTMLAttributes<HTMLAnchorElement>['referrerPolicy']
+}
+export type ParentLinkProps = PageHeaderProps & LinkProps
+
+const ParentLink = React.forwardRef<HTMLAnchorElement, ParentLinkProps>(
+  (
+    {
+      children,
+      sx = {},
+      href,
+      'aria-label': ariaLabel = `Back to ${children}`,
+      as = 'a',
+      hidden = hiddenOnRegularAndWide,
+    },
+    ref,
+  ) => {
+    const isHidden = useResponsiveValue(hidden, false)
+    return (
+      <>
+        <Link
+          ref={ref}
+          as={as}
+          aria-label={ariaLabel}
+          muted
+          sx={merge<BetterSystemStyleObject>(
+            {
+              display: isHidden ? 'none' : 'flex',
+              alignItems: 'center',
+              // min touch target size 44px (20 + (12x2))
+              // TODO: rem for line height?
+              // TODO: no 12px in theme - is that okay?
```

Comment:
If they in fact exist, we might be able to just set dimensions using the minimum touch target size tokens.

Reply:
I used only line-height to manage the min touch target size (2.75 rem). Let me know if you have any concerns!